MIGRATING_v2_to_v3.md

```diff
@@ -0,0 +1,223 @@
+   - `AuthorizationServerPolicyRuleConditions getConditions()`
+   - `Date getCreated()`
+   - `String getId()`
+   - `Date getLastUpdated()`
+   - `String getName()`
+   - `Integer getPriority()`
+   - `StatusEnum getStatus()`
+   - `Boolean getSystem()`
+   - `TypeEnum getType()`
+   - `AuthorizationServerPolicyRule setActions(AuthorizationServerPolicyRuleActions actions)`
+   - `AuthorizationServerPolicyRule setConditions(AuthorizationServerPolicyRuleConditions conditions)`
+   - `AuthorizationServerPolicyRule setName(String name)`
+   - `AuthorizationServerPolicyRule setPriority(Integer priority)`
+   - `AuthorizationServerPolicyRule setStatus(StatusEnum status)`
+   - `AuthorizationServerPolicyRule setSystem(Boolean system)`
+   - `AuthorizationServerPolicyRule setType(TypeEnum type)`
+   - `AuthorizationServerPolicyRule update(java.lang.String)`
+
+With the introduction of enum type `AuthorizationServerPolicyRule$StatusEnum`, operations `getStatus()` and `setStatus(StatusEnum status)` 
+will use this new enum type.
+   Enum `StatusEnum` has 2 fields listed below.
+   - `ACTIVE("ACTIVE")`
+   - `INACTIVE("INACTIVE")`
+
+### Package `com.okta.sdk.resource.authorization.server.policy.AuthorizationServerPolicyRuleActions`
+
+This is a newly created interface with methods listed below.
+   - `TokenAuthorizationServerPolicyRuleAction getToken()`
+   - `AuthorizationServerPolicyRuleActions setToken(TokenAuthorizationServerPolicyRuleAction token)`
+
+### Package `com.okta.sdk.resource.authorization.server.policy.AuthorizationServerPolicyRuleConditions`
+
+This is a newly created interface with methods listed below.
+   - `ClientPolicyCondition getClients`
+   - `GrantTypePolicyRuleCondition getGrantTypes()`
+   - `PolicyPeopleCondition getPeople()`
+   - `OAuth2ScopesMediationPolicyRuleCondition getScopes()`
+   - `AuthorizationServerPolicyRuleConditions setClients(ClientPolicyCondition clients)`
+   - `AuthorizationServerPolicyRuleConditions setGrantTypes(GrantTypePolicyRuleCondition grantTypes)`
+   - `AuthorizationServerPolicyRuleConditions setPeople(PolicyPeopleCondition people)`
+   - `AuthorizationServerPolicyRuleConditions setScopes(OAuth2ScopesMediationPolicyRuleCondition scopes)`
+   
+### Package `com.okta.sdk.resource.authorization.server.policy.TokenAuthorizationServerPolicyRuleAction`
+
+This is a newly created interface with methods listed below.
+   - `Integer getAccessTokenLifetimeMinutes()`
+   - `Integer getRefreshTokenLifetimeMinutes()`
+   - `Integer getRefreshTokenWindowMinutes()`
+   - `TokenAuthorizationServerPolicyRuleAction setAccessTokenLifetimeMinutes(Integer accessTokenLifetimeMinutes)`
+   - `TokenAuthorizationServerPolicyRuleAction setRefreshTokenLifetimeMinutes(Integer refreshTokenLifetimeMinutes)`
+   - `TokenAuthorizationServerPolicyRuleAction setRefreshTokenWindowMinutes(Integer refreshTokenWindowMinutes)`
+
+### Package `com.okta.sdk.resource.identity.provider.IdentityProvider`
+
+Below method has been removed.
```

Comment:
Please include a note on what the user that was using `deleteSigningKey` earlier should now do. Which method should the user start using now?

Reply:
It's an interesting question:) This method was removed [here](https://github.com/okta/openapi/commit/4b25db5ea48b14060c0e42886fd65d962908ae02)

Reply:
we should probably note: `deleteIdentityProviderKey` should be used instead.